lib/index.js

```diff
@@ -270,15 +299,24 @@ function getTokens(palette) {
 		widthRadioButton: "20px",
 		heightCheckbox: "20px",
 		widthCheckbox: "20px",
-		heightCountryFlag: "20px",
-		widthCountryFlag: "20px",
+		heightCountryFlag: "auto",
+		widthCountryFlag: "24px",
+		heightBadge: "24px",
+		widthBadgeCircled: "24px",
+		heightIllustrationSmall: "90px",
+		heightIllustrationMedium: "200px",
+		heightServiceLogoSmall: "12px",
+		heightServiceLogoMedium: "24px",
```

Comment:
Do we want to have this token named as `heightIllustrationMedium`?

Reply:
I think so, there was a discussion about Illustration sizes and I think that it will remain the same. Or do you think something else?